gym_ai2thor/image_processing.py

```diff
@@ -9,4 +9,4 @@ def rgb2gray(rgb):
     RGB image transformation to Luma component assuming BT.601 color.
     https://en.wikipedia.org/wiki/Luma_(video)
     """
-    return np.dot(rgb[..., :3], [0.299, 0.587, 0.114])
+    return np.expand_dims(np.dot(rgb[..., :3], [0.299, 0.587, 0.114]), axis=2)
```

Comment:
You have a (h, w, 3) image and want an output of (h, w, 1) so that you can use the same code for both grayscale and RGB computations. At least that was my idea, just adding the dimension at the end so it is more "standard"

Reply:
Before we were going from (h, w, 3) to (h, w)? 
If so, it might break some code elsewhere where I'm adding unsqueezes, we'll have to check

Reply:
open a bugfix branch for that on a3c. You should get an image of (h,w,1) so that checks can be done on the 3rd channel.

Reply:
ill just push do it on the new branch